File: game-server/src/main/java/org/forfun/mmorpg/game/database/user/dao/IdentityDao.java
package org.forfun.mmorpg.game.database.user.dao;

import org.forfun.mmorpg.game.database.user.entity.AccountEnt;
import org.springframework.data.jpa.repository.JpaRepository;

public interface IdentityDao extends JpaRepository<AccountEnt, Long> {

}

File: game-server/src/main/kotlin/org/forfun/mmorpg/game/database/user/dao/IdentityDao.kt
package org.forfun.mmorpg.game.database.user.dao

import org.forfun.mmorpg.game.database.user.entity.IdentityEnt
import org.springframework.data.jpa.repository.JpaRepository

interface IdentityDao : JpaRepository<IdentityEnt, Int>
